design-reviews/recipes/task-design-review-template.md

```diff
@@ -0,0 +1,50 @@
+# Your Task Design Title Here (prefix with DRAFT/WIP to indicate level of completeness)
+
+> When developing a design document for a new task it should contain a detailed design proposal demonstrating how it will solve the goals outlined below.  
+> Not all tasks require a design review but when they do it is likely that there many unknowns or the solution may be more complex.
+> The design should include diagrams, pseudo code, interface contracts as needed to provide a detailed understanding of the proposal.
+
+* [Task Name](http://link-to-task-work-item)
+* [Story Name](http://link-to-story-work-item)
+* Organization: [Organization]
+* Engagement: [Engagement]
+* Customer: [Customer]
```

Comment:
Is Customer different from Organization? Maybe just keeping one link for Project / Engagement: ?